components/datetime/src/provider/time_zones.rs

```diff
@@ -123,6 +125,26 @@ pub struct LocationsV1<'data> {
     pub pattern_partial_location: Cow<'data, DoublePlaceholderPattern>,
 }
 
+/// An ICU4X mapping to the CLDR timeZoneNames exemplar cities.
+/// See CLDR-JSON timeZoneNames.json for more context.
```

Comment:
Say something about how we only retain the ones that are different than the ones in `LocationsV1Marker`?